agent/pom.xml

```diff
@@ -94,6 +95,7 @@
       <plugin>
         <groupId>com.github.eirslett</groupId>
         <artifactId>frontend-maven-plugin</artifactId>
+        <version>0.0.27</version>
```

Comment:
This seems to be a very old version of the plugin, latest is 1.6, which is defined in the root pom.xml. It should't be necessary to specify the version here.